internal/grpcrand/grpcrand.go

```diff
@@ -31,6 +31,14 @@ var (
 	mu sync.Mutex
 )
 
+// Int implements rand.Int on the grpcrand global source.
+func Int() int {
+	mu.Lock()
+	res := r.Int()
+	mu.Unlock()
+	return res
```

Comment:
Optional, if you want to do a cleanup below:

```
mu.Lock()
defer mu.Unlock()
return r.Int()
```

Reply:
Done